src/estimagic/optimization/subsolvers/bounded_newton_quadratic.py

```diff
@@ -190,14 +203,36 @@ def compute_conjugate_gradient_step(
         )
 
     else:
-        step_inactive = minimize_trust_conjugate_gradient(
-            gradient_inactive,
-            hessian_inactive,
-            trustregion_radius,
-            gtol_abs=gtol_abs_conjugate_gradient,
-            gtol_rel=gtol_rel_conjugate_gradient,
-        )
-        step_norm = np.linalg.norm(step_inactive)
+        if conjugate_gradient_method == "standard":
```

Comment:
Maybe `standard` is not a good name because it sounds like this will be the default. I could live with `"cg"`, "`steihaug_toint"` and `"trsbox"` and think it also aligns with the names we use internally.